package/docker/keeweb.conf

```diff
@@ -31,5 +31,5 @@ server {
     listen 80 default_server;
     listen [::]:80 default_server;
     server_name _;
-    return 301 https://$host$request_uri;
+    return 301 https://$http_host$request_uri;
```

Comment:
I don't understand a bit, won't this be always set to 80 because the server is listening on port 80 only, as stated two lines above.

How does this work in your setup, where does the SSL port come from here?